conda_requirements.txt

```diff
@@ -3,7 +3,7 @@ joblib>=0.8
 numpy
 pandas
 ruamel.yaml
-scikit-learn==0.23.1
```

Comment:
FYI: While I'm suggesting relaxing the requirement in a couple other places, _here_ I don't think anything needs to be done since this is just for development dependencies.